### ISSUE TITLE
mean waveform improvements

### ISSUE THREAD

**Issue by bendichter** [flatironinstitute/neurosift#141]
It's great to see this new mean waveform view in Neurosift!

<img width="1728" alt="image" src="https://github.com/flatironinstitute/neurosift/assets/844306/ce4b7a19-84db-4116-85b6-90f7d7aa6d8e">

It's not clear what the checkboxes do. Could we add labels?

<img width="1728" alt="image" src="https://github.com/flatironinstitute/neurosift/assets/844306/1591eb5e-dbf6-4400-a5a0-7cfa1d022900">

The 4th checkbox moves the data over. It looks like the intention is to add a y axis but it's not showing up.

<img width="1728" alt="image" src="https://github.com/flatironinstitute/neurosift/assets/844306/b727dfa6-2e5d-4e94-8d36-1ba1097379d2">

The 5th checkbox makes green dots appear. It looks like this is intended to be the channels. Is there a way to interact with these channels?

**Comment by magland** [comment]
@bendichter I agree the checkbox options are very confusing in this context. These were all added to accommodate features that Alessio and AIND wanted to have. But in the context of neurosift, these are not all relevant. I think I'll disable some of them.

I agree labels would be good... and for that I think it would make sense to put these at the bottom instead of in the left panel, because then there will be more horizontal space.
